modules/ROOT/pages/amazon/amazon-dynamodb-connector-studio.adoc

```diff
@@ -0,0 +1,153 @@
+
+[[configure-input-source]]
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met. You can configure one of these input sources to use with Amazon DynamoDB Connector:
+
+* `HTTP Listener`, which initiates a flow each time it receives a request on the configured host and port
+* `Scheduler`, which initiates a flow when a time-based condition is met
+
+For example, to configure `HTTP Listener`, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP* > *Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Amazon DynamoDB Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Amazon DynamoDB* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it is a good practice to configure a global element that can be used by all instances of that connector in the app. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Amazon DynamoDB system. Amazon DynamoDB Connector supports both Basic authentication and Role authentication.
+
+To configure the global element for Amazon DynamoDB Connector, follow these steps:
+
+. Select the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the `Connector configuration` field to access the global element configuration fields.
+. On the *General* tab, configure Basic authentication or Role authentication, as described in the <<basic-authentication,Basic Authentication>> and <<role-authentication,Role Authentication>> sections.
++
+You can enter your authorization credentials in the global configuration properties or you can reference a configuration file that contains ANT-style property placeholders (recommended). For information, see xref:introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
```

Comment:
Consideration: should we reverse the order of the methods of authorization so that referencing the config. file is first, since it is the preferred method?
Consider also the previous comment about recommendations.